addons/l10n_au/data/localisation.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data noupdate="1">
+		<record id="base.AUD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.NZD" model="res.currency">
+			<field name="position">before</field>
+			<field name="active" eval="True"/>
+		</record>
+
+		<record id="base.JPY" model="res.currency">
```

Comment:
@mart-e

These are common trading currencies for Australian businesses.  At a minimum, we would need NZD installed, as New Zealand branches and/or subsidiaries and customers are very common.

For the others, I can see that it may create "noise" for users who don't use them.

In relation to the "position" of currency symbols, should we propose that as a change to base?  Australians and New Zealanders would rarely put the currency symbol after the value.